Bottom: 3cbaafcfd7817f84bc1f6fcc9d876fd07bdade4c
Top:    97a047b091cc01ac2fae78fe162837e326fe266c
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:44:07 +0000

Invalidate board contour when a via / pin is altered

The drill-holes of vias and pins can affect the board contour


---

diff --git a/src/polygon.c b/src/polygon.c
index 0accd67..e92f479 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1813,6 +1813,10 @@ ClearFromPolygon (DataType * Data, int type, void *ptr1, void *ptr2)
           strcmp (layer->Name, "route") == 0)
         Data->outline_valid = false;
     }
+  else if (type == PIN_TYPE || type == VIA_TYPE)
+    {
+        Data->outline_valid = false;
+    }
 
   if (type == POLYGON_TYPE)
     InitClip (PCB->Data, (LayerType *) ptr1, (PolygonType *) ptr2);
